### PR TITLE
Remove Ping

### LINKED ISSUE #85: Players view doesn't refreshed
Steps to reproduce:
1. As a Player 1 create new room
2. As a Player 2 join this room

Result:
Player 2 can see both players in sidebar, but Player 1 doesn't see second player. They will only get visible after page reload

Expected result:
Player 1 can see new players as soon as they join

### LINKED ISSUE #86: Current player isn't highlighted
After recent update we no longer see "(You)" near our username

### PR DESCRIPTION
**Pull request by pastDexter**
- [x] Remove necessity for sending `Ping` every second.
- [x] Players connection status management moved to boardgame.io fork (plans to push to upstream after we validate if it's working properly)
- [x] move and refactor players management to `BoardGameContext`
- [x] Move turn timers to client side

Rememeber to reinstall dependencies `npm install`

Closes #85 
Closes #86

### CODE REVIEW

File: src/games/kalambury/Drawing.js [hunk 1]
```diff
@@ -35,7 +32,7 @@ const Drawing = ({ lines, drawable, ...props }) => {
           <DrawingLine key={id} line={scaleToViewBox(line)} viewBoxWidth />
         ))}
       </svg>
-      <Progress indicating percent={(100 * G.remainingSeconds) / 120} attached="bottom" />
+      <Progress indicating percent={(100 * remainingSeconds) / 120} attached="bottom" />
```
Comment (Bombasarkadian) [inline src/games/kalambury/Drawing.js:35]: Maybe we shouldn't have this `120` as a constant here? 🤔

File: .storybook/BoardGameContextMock.js [hunk 2]
```diff
@@ -1,57 +1,62 @@
...
-        },
+const BoardGameContextMock = ({ children }) => {
+  return (
+    <BoardGameProvider
+      G={{
+        timePerTurn: 120,
+        turnEndTime:
+          Math.floor(new Date().getTime() / 1000) +
+          number(
+            "Turn End Time",
```
Comment (Bombasarkadian) [inline .storybook/BoardGameContextMock.js:16]: I think "Remaining seconds" was a good description for this knob

File: src/components/game/Sidebar/Player.js [hunk 3]
```diff
@@ -53,7 +54,7 @@ const Player = ({
                 duration={200}
                 verticalAlign="middle"
               >
-                {actions.slice(0, 3).map((action, idx) => (
```
Comment (Bombasarkadian) [inline src/components/game/Sidebar/Player.js:56]: Are you sure about this? 
Screenshot 2020-04-08 at 17 53 33

File: src/games/kalambury/Kalambury.js [hunk 4]
```diff
@@ -174,48 +144,41 @@ export const Kalambury = {
...
+          if (currentTime() >= G.turnEndTime) {
             LogAction(G, ctx, ctx.currentPlayer, "timeout", { previous: G.secret.phrase }, true);
             G.points[ctx.currentPlayer] -= 1;
           }
         },
-        endIf: (G, _ctx) => G.remainingSeconds <= 0,
+        endIf: (G, _ctx) => currentTime() >= G.turnEndTime,
         order: {
           first: () => 0,
           next: (G, ctx) => {
```
Comment (Bombasarkadian) [inline src/games/kalambury/Kalambury.js:160]: It doesn't seem to work. I've created 2 people room, joined both slots, started the game. After the 2nd player guesses correctly the turn ends (points are added and phrase changes), but the 1st player is still drawing

File: src/components/game/Sidebar/Player.js [hunk 5]
```diff
@@ -33,14 +34,14 @@ const Player = ({
...
-    <Segment disabled={!isActive && false}>
+    <Segment disabled={!isConnected && false}>
       <Feed>
         <Feed.Event>
           <Feed.Label image={photoURL} />
           <Feed.Content>
             <Feed.Date>
               {displayName} {isCurrentPlayer && <span>({t("sidebar.player.current")})</span>}{" "}
-              {!isActive && <span>({t("sidebar.player.disconnected")})</span>}
+              {!isConnected && <span>({t("sidebar.player.disconnected")})</span>}
```
Comment (Bombasarkadian) [inline src/components/game/Sidebar/Player.js:44]: Maybe we could gray-out disconnected players?

File: src/components/game/Sidebar/Player.js (hunk 3 above)
Comment (pastDexter) [reply src/components/game/Sidebar/Player.js:56]: I removed it for debugging 🙈

File: src/components/game/Sidebar/Player.js (hunk 5 above)
Comment (pastDexter) [reply src/components/game/Sidebar/Player.js:44]: it marks them as disconnected as soon as they disconnect from the socket, so even if you refresh the page, you will be disconnected for a second, so graying-out may introduce some flickering

File: .storybook/BoardGameContextMock.js (hunk 2 above)
Comment (pastDexter) [reply .storybook/BoardGameContextMock.js:16]: 👍

File: src/games/kalambury/Board/GameBoard.js [hunk 6]
```diff
@@ -23,24 +25,34 @@ const confettiConfig = {
...
   const [lastSuccess, setLastSuccess] = useState(false);
   const lastUserGuess = filterActions(G.actions, playerID, "guess")[0] || {
     id: null,
     success: false,
   };
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const seconds = G.turnEndTime - currentTime();
+      if ((seconds <= 0 && isDrawing) || seconds < 5) {
```
Comment (Bombasarkadian) [inline src/games/kalambury/Board/GameBoard.js:48]: Should it really be `seconds < 5`? This condition will be met before the first one every time

Comment (pastDexter) [reply src/games/kalambury/Board/GameBoard.js:48]: no, it should be `-5` Good catch 👍

File: src/index.css [hunk 7]
```diff
@@ -40,6 +40,11 @@ code {
   left: 50%;
 }
 
+.smooth-disabled {
+  transition: opacity 2s cubic-bezier(0.55, 0.06, 0.82, 0.18),
```
Comment (Bombasarkadian) [inline src/index.css:44]: We should only have slow animation for disconnecting players. Connecting ones shouldn't have 2s animation

**Pull request merged**